src/sonic-config-engine/minigraph.py

```diff
@@ -645,27 +649,69 @@ def parse_spine_chassis_fe(results, vni, lo_intfs, phyport_intfs, pc_intfs, pc_m
 #
 ###############################################################################
 
-def filter_acl_mirror_table_bindings(acls, neighbors, port_channels):
+def filter_acl_table_bindings(acls, neighbors, port_channels, sub_role):
+    filter_acls = {}
+    """
+        If the asic role is BackEnd no ACL Table (Ctrl/Data/Everflow) is binded.
+        This will be applicable in Multi-NPU Platforms.
     """
-        Filters out inactive front-panel ports from the binding list for mirror
-        ACL tables. We define an "active" port as one that is a member of a
-        port channel or one that is connected to a neighboring device.
+    if sub_role == 'BackEnd':
+        return filter_acls
+
+    front_port_channel_intf = []
+   
     """
+        Get the front panel port channel. It will use port_alias_asic_map
+        which will get populated from port_config.ini for Multi-NPU 
+        architecture
+    """ 
+    for port_channel_intf in port_channels:
+        front_port_channel = True
+        for port_channel_member in port_channels[port_channel_intf]['members']:
+            if port_channel_member in port_alias_asic_map and port_channel_member.startswith(backend_asic_interface_name_prefix):
+                front_port_channel = False
+                break
+        if front_port_channel == True:
+            front_port_channel_intf.append(port_channel_intf)
```

Comment:
It took me a bit to understand what was going on here, I think something like this might be more clear:
```
backend_port_channel = any(lag_member in port_alias_asic_map \
                           and lag_member.startswith(backend_asic_interface_name_prefix) \
                           for lag_member in port_channels[port_channel_intf]['members'])
if not backend_port_channel:
    front_port_channel_intf.append(port_channel_intf)
```

IMO this reads closer to what you're doing, but I don't know how familiar everyone is with this syntax. @jleveque @lguohan what do you think?

Reply:
@daall will look into it.

Reply:
@daall@microsoft.com Thanks for input . Updated.

---
In reply to: [421175854](https://github.com/Azure/sonic-buildimage/pull/4491#discussion_r421175854) [](ancestors = 421175854)